.github/workflows/publish_charm.yaml

```diff
@@ -47,26 +113,38 @@ jobs:
           cp -rp $TEMP_DIR/. .
           rm -rf $TEMP_DIR
       - name: Install charmcraft
-        run: sudo snap install charmcraft --classic
-      - name: Log in to the Container registry
-        uses: docker/login-action@v2
-        with:
-          registry: ${{ env.REGISTRY }}
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo ${{ needs.get-commit-sha.outputs.sha }}
+          sudo snap install charmcraft --classic
+      - name: Download image artifacts
+        run: |
+          gh api \
+            --method GET \
+            -H "Accept: application/vnd.github+json" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            /repos/${{ github.repository }}/actions/runs \
+            -f status=${{ needs.get-commit-sha.outputs.wf-status }} \
+            -f head_sha=${{ needs.get-commit-sha.outputs.sha }} \
+            -f event=pull_request \
+            --jq '[.workflow_runs[] | select(.path == ".github/workflows/integration_test.yaml") | .id]' | read -r RUN_IDS
```

Comment:
Is RUN_IDS actually getting the output? Maybe doing the other way around?
Something like read RUN_IDS < <(echo "ABC")

Reply:
It isn't. I'm trying some things out. Thanks for your input :)